tests/filesTest.js

```diff
@@ -1381,11 +1521,14 @@ describe('Main: Currently testing files management,', function () {
     })
 
     it('checking method: create tag', async function () {
-      const newTagName = 'testSystemTag12345'
       const provider = createProvider(true, true)
```

Comment:
should pass now in oc10, can you set it to `const provider = createProvider(false, true)`